cmd/kogito/command/deploy/delete_service_test.go

```diff
@@ -35,7 +35,7 @@ func Test_DeleteServiceCmd_SuccessfullyDelete(t *testing.T) {
 
 	lines, _, err := test.ExecuteCli()
 	assert.NoError(t, err)
-	assert.Contains(t, lines, "Successfully deleted Kogito Service example-drools")
+	assert.Contains(t, lines, "Successfully deleted")
```

Comment:
I have noticed that some log entriesin CLI are adjusted, for example:
```
[karel@localhost kogito-cloud-operator]$ kogito new-project ksuta-test
Successfully created	{"Project": "ksuta-test"}
[karel@localhost kogito-cloud-operator]$ kogito deploy-service test
Creating resource	{"kind": "", "name": "test", "namespace": "ksuta-test"}
Kogito Build Service successfully installed in the Project ksuta-test.
Check the Service status by running 'oc describe kogitobuild/test -n ksuta-test'
Your Kogito Runtime Service needs the application binaries to proceed. To upload your binaries please run 'oc start-build test --from-dir=target -n ksuta-test' from your project's root
Creating resource	{"kind": "", "name": "test", "namespace": "ksuta-test"}
Kogito Service successfully installed in the Project ksuta-test.
Check the Service status by running 'oc describe kogitoruntime/test -n ksuta-test'
To more easily manage your Kogito Service install Data Index Service and Process Instance Management. 
For how to install see: https://docs.jboss.org/kogito/release/latest/html_single/#con-kogito-operator-with-data-index-service_kogito-deploying-on-openshift
and https://docs.jboss.org/kogito/release/latest/html_single/#con-management-console_kogito-developing-process-services
```
I found it a bit hard to understand what actually happened with this JSON format logs, the previous approach with natural language was more readable. Is there a specific reason for using such JSON format? Or is it a preferred approach now?

Reply:
This new format kind of JSON is the new preferred approach and is the only way to define logs with the logger implementation comes with the operator-sdk. We need to define everything in `"key", "value"` so formatted logs are not allowed, There is a trick for using the formatted logs that is used to print logs like this one 
```
Your Kogito Runtime Service needs the application binaries to proceed. To upload your binaries please run 'oc start-build test --from-dir=target -n ksuta-test' from your project's root
``` since it requires some addition in the middle but it is not recommended. For more about [this logger see](https://github.com/go-logr/logr)

Reply:
hmm, ok

Reply:
@Kaitou786 I don't think this JSON approach is a good solution for the CLI. I'd rather get back to the old format for the CLI. In this case we can drop the `klog` module to use `loggrus` instead. The last one is used by minikube and CRC as well.

Reply:
Seeing loggrus is also a good option but that too doesn't fullly support the formatted(human friendly logging) log that we would want to use with our CLI, [see](https://github.com/sirupsen/logrus#level-logging). Currently we switched to our earlier zap logger for the formatted logging for not changing too much.

Reply:
> Seeing loggrus is also a good option but that too doesn't fullly support the formatted(human friendly logging) log that we would want to use with our CLI, [see](https://github.com/sirupsen/logrus#level-logging). 

I'm not sure if I follow. Why is it not a good option for human-friendly logging? Using loggrus I can see an output like:

```
INFO Checking if podman remote executable is cached 
INFO Checking if goodhosts executable is cached   
INFO Checking minimum RAM requirements            
INFO Checking if running as non-root              
INFO Checking if Virtualization is enabled        
INFO Checking if KVM is enabled                   
INFO Checking if libvirt is installed             
INFO Checking if user is part of libvirt group    
INFO Checking if libvirt daemon is running 
```

Doesn't this help? Also, it's used by crc and operator-sdk. We can relate and even use with BDD since they can output to files as well.

But seeing this disclaimer:

> I believe Logrus' biggest contribution is to have played a part in today's widespread use of structured logging in Golang. There doesn't seem to be a reason to do a major, breaking iteration into Logrus V2, since the fantastic Go community has built those independently. Many fantastic alternatives have sprung up. Logrus would look like those, had it been re-designed with what we know about structured logging in Go today. Check out, for example, Zerolog, Zap, and Apex.

I believe that we should stick to Zap.